### ISSUE TITLE
[attention_ocr] about eval.py and demo_inference.py different reslut

### ISSUE THREAD

**Issue by ccnankai** [tensorflow/models#2919]
There are only numbers in my training samples.
 When I run #  python eval.py --split_name test --train_log_dir , it shows good results, 
![webwxgetmsgimg](https://user-images.githubusercontent.com/33537489/33356004-0780b9ba-d4f5-11e7-8613-685589304c5e.jpg)
accuracy get 0.99.

but wehn I  predict one picture only , converted the image to [150,600,3], 
python demo_inference.py –batch_size=1 –checkpoint= 
--image_path_pattern=/home..../pic/%02d.png ,None of the predicted strings are right.
 ;;;;;;;
![webwxgetmsgimg 1](https://user-images.githubusercontent.com/33537489/33356013-15702830-d4f5-11e7-8899-c9c1a3b3f778.jpg)

This a problem that puzzled me for a long time.
can you tell me the reason,thank you very much.

@bmabey @alextp @moonboots

**Comment by ccnankai** [comment]
Sorry, I should go to stackoverflow to ask questions.
